core/src/main/scala/org/apache/spark/scheduler/ExecutorLossReason.scala

```diff
@@ -58,3 +58,11 @@ private [spark] object LossReasonPending extends ExecutorLossReason("Pending los
 private[spark]
 case class SlaveLost(_message: String = "Slave lost", workerLost: Boolean = false)
   extends ExecutorLossReason(_message)
+
+/**
+ * A loss reason that means the executor is marked for decommissioning.
+ *
+ * This is used by the task scheduler to remove state associated with the executor, but
+ * not yet fail any tasks that were running in the executor before the executor is "fully" lost.
+ */
+private [spark] object ExecutorDecommission extends ExecutorLossReason("Executor Decommission.")
```

Comment:
A very nitpicky, not code related thing (sorry!), but shouldn't this reason be "Executor decommission" (lowercase D), to be consistent with the other reason strings in the file? Maybe there is a reasoning behind I don't see though.

Reply:
Thanks :)